iframe/priv/static/iframe/v5.html

```diff
@@ -102,6 +102,14 @@
             state.secretHandler = callback;
             postMessage({ type: "selectSecret", preselectName, options });
           },
+
+          setSmartCellEditorIntellisenseNode(node, cookie) {
+            postMessage({
+              type: "setSmartCellEditorIntellisenseNode",
+              node,
+              cookie,
```

Comment:
I'm wondering if we should let the smart cell set the cookie and assume we can connect. @josevalim wdyt?

Reply:
I think we need to receive the cookie because connections are not transitive on hidden nodes (which our runtimes are). Especially not when the cookie changes along the way.

We also need to investigate what happens to all of our erpc calls when the node does not exist. :) But that's a separate convo.

Reply:
Intelligence runs within the runtime node :)

Reply:
So you are right, we don't need a cookie!

Reply:
There may be an advantage of having a cookie though, it means the information will be there if we need it. So do we want to be safer than sorry?

Reply:
I don't have a strong preference on passing it or not, but for intellisense I'm leaning towards assuming we are connected, rather than having the cookie logic there too.

Reply:
Well, doesn't hurt to pass, we can even totally ignore it for now if we want. Could be a bit confusing, but we are likely the only years of this, so that's fine :D

Reply:
Yeah, I would pass it, just because it gives more flexibility, even if we discard it.

The only concern with passing it is if someone has access to the cookie value in the JavaScript code, but all JavaScript code is isolated in iframes, and they will have access to the node via the Elixir runtime anyway.

Reply:
Alright, so pls approve if you are happy :D